e2e/poms/task1.ts

```diff
@@ -0,0 +1,190 @@
+        await this.page.getByTestId(TASK1SELECTOR.testIds.startFromScratchButton).click();
+    }
+    
+    async buildFor1() {
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addFullNameElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.fullNamePreviewGroup)).toBeVisible();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addPhoneNumberElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.phonePreviewGroup)).toBeVisible();
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishButton).click();
+        const previewPromise = this.page.waitForEvent("popup");
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishPreviewButton).click();
+        const page2 = await previewPromise;
+        return page2;
+    }
+
+    async submit(page2: Page) {
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.phoneGroup)).toBeVisible();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.fullNameGroup)).toBeVisible();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.emailGroup)).toBeVisible();
+        await page2.getByTestId(TASK1SELECTOR.testIds.startOrSubmitButton).click();
+        await expect(page2.getByText(TASK1TEXT.texts.phoneNumberInvalid)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.firstNameRequired)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.lastNameRequired)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.emailRequired)).toBeVisible();
+
+        await page2.getByTestId(TASK1SELECTOR.testIds.emailTextField).fill(TASK1TEXT.texts.email)
+        await page2.getByTestId(TASK1SELECTOR.testIds.firstNameTextField).fill(TASK1TEXT.texts.firstName)
+        await page2.getByTestId(TASK1SELECTOR.testIds.lastNameTextField).fill(TASK1TEXT.texts.lastName)
+        await page2.getByTestId(TASK1SELECTOR.testIds.phoneNumberInputField).fill(TASK1TEXT.texts.phoneNo)
+        await page2.getByTestId(TASK1SELECTOR.testIds.startOrSubmitButton).click();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.thankYouPageContent)).toBeVisible();
+
+        await this.page.bringToFront();
+        await this.page.waitForTimeout(TASK1SELECTOR.timeouts.short);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.submissionsTab).click();
+        await expect(this.page.getByText(TASK1TEXT.texts.email)).toBeVisible();
+    }
+
+    async deleteForm() {
+        await this.page.bringToFront();
+        await this.page.locator(`//button[@data-testid='${TASK1SELECTOR.testIds.neetoMoleculesMenuButton}']`).click()
+        await this.page.getByTestId(TASK1SELECTOR.testIds.formDeleteButton).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.deleteArchiveAlertArchiveCheckbox).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.deleteArchiveAlertDeleteButton).click();
+    }
+
+    async buildFor2() {
```

Comment:
Always use arrow functions.
```suggestion
    buildFor2 = async () => {
```